build/sdk/js/templates/api_doc.mustache

```diff
@@ -32,9 +32,7 @@ const sstk = require('shutterstock-api');
 {{#authMethods}}{{#isBasic}}
 const apiClient = sstk.ApiClient.instance;
```

Comment:
We no longer need the line `const apiClient = sstk.ApiClient.instance;` here.

Reply:
Nice, thanks!